cypress/support/modularTransportClient/transportLayers/websocket.js

```diff
@@ -81,7 +81,25 @@ export default class Websocket extends AsyncTransportClient {
         res(event.data);
       };
 
-      this.ws.addEventListener('message', sendCallback);
+      this.ws.onmessage = (msg) => {
+        const message = JSON.parse(msg.data);
+        const requestMessage = JSON.parse(payload);
```

Comment:
// Updating the listenerResponse in the MAP when event response received.
to be changed to 
        // Updating event response in the map when event is triggered by the platform

Reply:
addressed the comment